src/components/food-selection/restaurantsCards.js

```diff
@@ -1,18 +1,47 @@
-import React from 'react'
+import React, {useState} from 'react';
 import RestaurantCard from "./restaurantCard";
+
+import Col from "react-bootstrap/Col";
+
 import { allData } from "./allData";
 
 
 const RestaurantsCards = () => {
 // () can replace return 
 // "return" returns a html / UI
+
+    const [currentPage, setCurrentPage] = useState(0);
+    const currentRestaurant = allData.data[currentPage];
+    const totalRestaurants = allData.data.length;
+
+    function nextPage() {
+        if (currentPage < totalRestaurants - 1) {
+            setCurrentPage(currentPage + 1);
+        }
+    }
+
     return (
         <>        
-        {
-        allData.data.map( (restaurant, id)=> (
-        <RestaurantCard key={id} restaurant={restaurant}></RestaurantCard>
-        ))
-        }
+        
+        <RestaurantCard restaurant = {currentRestaurant}></RestaurantCard>
+        <Col>
+            <button onClick = {nextPage}> Next
```

Comment:
hmm, do we need a next button? i thought it should be tinder style, where it goes to the next page after the user swipes left or right

Reply:
because we arent building an app but instead a web app, we will definitely need the next button. swiping on web app is not a common gesture control. Also for implementation ease we should just go with tapping instead.

Reply:
oh, sorry i wasn't referring to the tapping/swiping.
rather, i meant that - shouldn't the "ok/not ok" button be used to take them to the next page? do we need another "next" button for that?